common/language.c

```diff
@@ -264,7 +264,7 @@ const char* const g_strings[StrId_Max][16] =
         STR_EN("Open"),
         STR_ES("Abierto"),
         STR_DE("Öffnen"),
-        STR_FR("Ouvrir"),
+        STR_FR("Ouvrir"),   //if "Open" is used as an adjective here (just like Spanish), and not a verb, consider changing "Ouvrir" to "Ouvert", because "Ouvrir" is the verb "To open"
```

Comment:
This is a verb. Spanish translation should have been `Abrir`...

Reply:
Thanks, then STR_IT and STR_PT are probably wrong too. I'm not fluent/confident enough to make the changes for those two. But I'll do STR_ES then.